packages/esm-patient-list-app/src/add-patient/add-patient.component.tsx

```diff
@@ -111,10 +86,22 @@ const AddPatient: React.FC<AddPatientProps> = ({ closeModal, patientUuid }) => {
           value={searchValue}
         />
       </div>
+      {Object.keys(error).find((key) => {
+        console.error(error[key]);
+        return error[key];
+      }) && (
+        <div>
+          <InlineNotification
+            title="Error Loading lists"
+            subtitle="An error occurred while loading patient lists. Check the dev tools for more info"
+            hideCloseButton
+          />
+        </div>
+      )}
```

Comment:
this fires up an inline error notification if one of the errors occurs and then logs more error info to the console. 
![Screenshot from 2023-05-03 08-59-11](https://user-images.githubusercontent.com/104269786/235843490-7da2cfed-6e0f-4144-a714-dacf945523b0.png)

Reply:
If in case more than one errors occur. The first reachable one is logged into the console